examples/experimental/onnx/classification/onnx_ptq_classification.py

```diff
@@ -41,12 +41,8 @@ def run(onnx_model_path: str, output_model_path: str,
     if input_shape is None:
         nncf_logger.info(
             "input_shape is None. Infer input_shape from the model.")
-        input_shape = infer_input_shape(original_model)
 
-    input_keys = [node.name for node in original_model.graph.input]
-    if len(input_keys) != 1:
-        raise RuntimeError(
-            f"The number of inputs should be 1(!={len(input_keys)}).")
+    input_shape, input_keys = infer_input_shape(original_model, input_shape)
```

Comment:
I recommend you to solve this problem by adding an input argument of input tensor name rather than picking up the input tensor automatically. We can ask user to give the proper input tensor name if we fail to find the input tensor.

Reply:
I agree. I fixed to ask user either the input tensor's name or shape and to find the proper tensor shape and name.